CharacterCreation/03_Feats.txt

```diff
@@ -507,7 +549,59 @@ result in extra hit points, or take the average of the result (1d10 = 5,
 2d10 = 10, can be taken twice). If any of your die rolls is a 1 or 2, reroll
 the die.
 
+== Unarmed Combat Training ==
+
+Prerequisites:
+* STR 5 and DEX 5 and Level 5, OR
+* Assassin Class, OR
+* Berserker Class
+
+	Increase base and minimum damage of Unarmed Strike by 20. 
+	
+	Additionally, Once per round after a successful melee strike, 
+if the target is not wielding a CQB weapon, you may use a reaction to 
+make a disarm attempt.
+
+== Unarmed Combat Expertise ==
+
+Prerequisites:
+* STR 6 and DEX 6 and Level 10, OR
+* Assassin Class and Level 5, OR
+* Berserker Class and Level 5
+
+	Increase base and minimum damage of Unarmed Strike by 20.

```

Comment:
Is this an additional 20 on top of the extra damage from UACT, or is it the same one?

Reply:
Additional...I just noticed that the feats don't have themselves as prereqs. It seems like Expert should require Training.